src/common/metrics/hospitalizations.ts

```diff
@@ -23,30 +23,30 @@ export const HOSPITAL_USAGE_LEVEL_INFO_MAP: LevelInfoMap = {
     upperLimit: LIMIT_LOW,
     name: LOW_NAME,
     color: COLOR_MAP.GREEN.BASE,
-    detail: SHORT_DESCRIPTION_LOW,
+    detail: () => SHORT_DESCRIPTION_LOW,
   },
   [Level.MEDIUM]: {
     level: Level.MEDIUM,
     upperLimit: LIMIT_MEDIUM,
     name: MEDIUM_NAME,
     color: COLOR_MAP.ORANGE.BASE,
-    detail: SHORT_DESCRIPTION_MEDIUM,
+    detail: () => SHORT_DESCRIPTION_MEDIUM,
   },
   [Level.HIGH]: {
     level: Level.HIGH,
     upperLimit: LIMIT_HIGH,
     name: HIGH_NAME,
     color: COLOR_MAP.RED.BASE,
-    detail: SHORT_DESCRIPTION_HIGH,
+    detail: () => SHORT_DESCRIPTION_HIGH,
   },
   [Level.UNKNOWN]: {
     level: Level.UNKNOWN,
     upperLimit: 0,
     name: UNKNOWN,
     color: COLOR_MAP.GRAY.BASE,
-    detail: SHORT_DESCRIPTION_UNKNOWN,
+    detail: () => SHORT_DESCRIPTION_UNKNOWN,
   },
 };
 
 export const HOSPITALIZATIONS_DISCLAIMER =
```

Comment:
this should just contain the link..

Reply:
Yeah I wasn't sure how to do that, without something like `dangerouslySetInnerHTML={createMarkup()} `